docs/ChatterinoTheme.schema.json

```diff
@@ -390,6 +390,11 @@
           "$comment": "Determines which icons to use. 'dark' will use dark icons (best for a light theme). 'light' will use light icons.",
           "enum": ["light", "dark"],
           "default": "light"
+        },
+        "fallbackTheme": {
+          "$comment": "Determined which built-in Chatterino theme to use as a fallback in case a color isn't configured.",
```

Comment:
The comment above uses "Determines", this uses "Determined".

Reply:
Fixed in a force push - I never made this typo this must've been a bug in GitHub's interface :)